### PR TITLE
Moving towards production readiness

### PR DESCRIPTION
**Pull request by codepoet2017390**
- Made production ready UI tweaks and added all screen size compatibility

- Minimal code tweaks for better adherence to architecture

### CODE REVIEW

**Comment by HerbertYiga** [comment]
could you squash your commits here!!

File: app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt [hunk 1]
```diff
@@ -45,7 +47,7 @@ class RegisterPatientActivity : AppCompatActivity() {
     fun setUpUI() {
         setSupportActionBar(binding.toolbar)
         val actionBar = supportActionBar
-        actionBar!!.title = getString(R.string.appointmentFormheading)
+        actionBar!!.title = "consult a doctor"
```
Comment (isears) [inline app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt:50]: Let's go ahead and change the name of the "Register Patient" button on the homescreen to match the title of this activity. Also let's do title case capitalization.

File: app/src/main/res/layout/splash_screen.xml [hunk 2]
```diff
@@ -1,67 +1,122 @@
...
-             />
+        android:id="@+id/imageView"
+        android:layout_width="300dp"
+        android:layout_height="300dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="8dp"
```
Comment (isears) [inline app/src/main/res/layout/splash_screen.xml:21]: Is there a way to specify all margins 8dp instead of specifying each individually?

**Review by isears — CHANGES_REQUESTED** [review]
Looking good, just need a few minor tweaks!

File: app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt (hunk 1 above)
Comment (codepoet2017390) [reply app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt:50]: Yup, that's a really nice idea.

File: app/src/main/res/layout/splash_screen.xml (hunk 2 above)
Comment (codepoet2017390) [reply app/src/main/res/layout/splash_screen.xml:21]: Ok will do that if possible

**Comment by isears** [comment]
> could you squash your commits here!!

@HerbertYiga this is a good point. We'll make sure to use the github 'squash and merge' functionality when the PR gets merged.

File: app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt (hunk 1 above)
Comment (codepoet2017390) [reply app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt:50]: Done

File: app/src/main/res/layout/splash_screen.xml (hunk 2 above)
Comment (codepoet2017390) [reply app/src/main/res/layout/splash_screen.xml:21]: I tried doing that but I don't know why UI distorts on smaller screens so let's stay with this as of now

**Comment by codepoet2017390** [comment]
@isears  Updated.

**Review by isears — APPROVED** [review]
Alright let's squash & merge

**Comment by codepoet2017390** [comment]
@HerbertYiga @isears  Squashed and merged

**Pull request merged**